mizar/common/rpc.py

```diff
@@ -165,15 +173,20 @@ def update_ep(self, ep):
         cmd = f'''{self.trn_cli_update_ep} \'{jsonconf}\''''
         logger.info("update_ep: {}".format(cmd))
         returncode, text = run_cmd(cmd)
-        logger.info("returns {} {}".format(returncode, text))
+        logger.info("update_ep {} returns {} {}".format(cmd, text, returncode))
+        if (CONSTANTS.RPC_ERROR_CODE in text or
+            CONSTANTS.RPC_FAILED_CODE in text or
+                CONSTANTS.RPC_FATAL_CODE in text):
+            task.raise_temporary_error(
+                "update ep returned ERROR! Retrying as agent may have not yet been loaded.")
```

Comment:
Can we please make it consistent? Use "update_ep" (the function name for easier error grepping)

Reply:
Will do